continuous-delivery/readme.md

```diff
@@ -141,3 +141,15 @@ Try to limit the number of versions of your application running parallel in prod
 * [Continuous Delivery](https://www.continuousdelivery.com/) by by Jez Humble, David Farley.
 * [Continuous integration vs. continuous delivery vs. continuous deployment](https://www.atlassian.com/continuous-delivery/principles/continuous-integration-vs-delivery-vs-deployment)
 * [Deployment Rings](https://docs.microsoft.com/en-us/azure/devops/migrate/phase-rollout-with-rings?view=azure-devops)
+
+
+### Tools
+
+Check out the below tools to quickly get some of these CD best practices:
```

Comment:
These are not known best practices.  Consider rewording.

Reply:
done